react/week2/TodoDisplay.js

```diff
@@ -0,0 +1,58 @@
+import './App.css';
+import React,{useState} from "react";
+let todos = [
+    {
+      id: 1,
+      description: "Get out of bed",
+      isDone: false
+    },
+    {
+      id: 2,
+      description: "Brush teeth",
+      isDone: false
+    },
+    {
+      id: 3,
+      description: "Eat breakfast",
+      isDone: false
+    },
+    {
+      id: 4,
+      description: "random text",
+      isDone: false
+    },
+   ];
+  export function TodoDisplay(){
+    const [state, setState] = useState([]);
+    
+    const addTodo = () => {
+    const randomNumber =Math.floor((Math.random() * todos.length) );
+       setState(prev => {
+        return [...prev, {
+        id : todos[todos.length -1]?.id + 1 || 1,
+        description :todos[randomNumber].description ,
+        isDone : false
+      }
+      
+    ]})
+      console.log(state);
```

Comment:
Just for the best practice, Always remember to remove console log from your final code push.